Makefile

```diff
@@ -15,19 +15,21 @@ include config.public.mk
 endif
 
 RUN = uv run
-SCHEMA_NAME = $(LINKML_SCHEMA_NAME)
-SOURCE_SCHEMA_PATH = $(LINKML_SCHEMA_SOURCE_PATH)
-SOURCE_SCHEMA_DIR = $(dir $(SOURCE_SCHEMA_PATH))
 SRC = src
 DEST = project
-PYMODEL = $(SRC)/$(SCHEMA_NAME)/datamodel
 DOCDIR = docs
-DOCTEMPLATES = $(SRC)/docs/templates
 EXAMPLEDIR = examples
+SCHEMA_NAME = $(LINKML_SCHEMA_NAME)
```

Comment:
minor reorganisation of the directory structure and addition of a couple of variables